apps/ewallet/lib/ewallet/web/paginator.ex

```diff
@@ -7,6 +7,7 @@ defmodule EWallet.Web.Paginator do
   alias EWalletDB.Repo
 
   @default_per_page 10
+  @default_max_per_page 100
```

Comment:
Why did you decide to add this `@default_max_per_page` even though we already define a default value in the config? 🤔

Reply:
A lot of the libraries have defaults at the library level. I think this guarantees that even if the values are missing from the config, the system will still work, so I want to follow this pattern more. E.g:

https://github.com/phoenixframework/phoenix/blob/master/lib/phoenix/endpoint/supervisor.ex#L111-L132

https://github.com/mschae/cors_plug/blob/master/lib/cors_plug.ex#L4-L16

https://github.com/getsentry/sentry-elixir/blob/master/lib/sentry/config.ex#L8-L15